unit_tests/test-framework/engine_test_helper.cpp

```diff
@@ -401,10 +401,10 @@ const AngleBasedEvent * EngineTestHelper::assertTriggerEvent(const char *msg,
 	auto event = engine.module<TriggerScheduler>()->getElementAtIndexForUnitTest(index);
 
 	if (callback) {
-		assertEqualsM4(msg, " callback up/down", (void*)event->action.getCallback() == (void*) callback, 1);
+		EXPECT_EQ(1, reinterpret_cast<void*>(event->action.getCallback()) == reinterpret_cast<void*>(callback)) << " callback up/down";
```

Comment:
why `EXPECT_EQ(1, a == b)` instead of `EXPECT_EQ(a, b)`?

Reply:
fixed, thanks :D